common/tools/dev-tool/register.js

```diff
@@ -48,53 +42,14 @@ if (process.env.DEBUG) {
 
 require("dotenv").config();
 
-/**
- * The transformer we will use to handle imports of the host package.
- *
- * For example, if an @azure/template sample needs to import @azure/template,
- * this transformer will handle rewriting the imports for us.
- */
-const makeTransformers = () => ({
-  before: [
-    (transformationContext) => (sourceFile) =>
-      ts.visitEachChild(
-        sourceFile,
-        (node) => {
-          // If the sample or test program is trying to import the host
-          // package, we'll rewrite it on the fly to make the import work.
-          if (ts.isImportDeclaration(node) && node.moduleSpecifier.text === packageNameToPatch) {
-            // rewrite the import to use a relative path
-            const oldName = node.moduleSpecifier.text;
-            const base = sourceFile.path.includes("dist-esm") ? path.join(cwd, "dist-esm") : cwd;
-            node.moduleSpecifier.text = path.relative(
-              // This is marked internal in the TS API, need to make sure there's not a better way
-              // to get the path from the sourceFile node
-              path.dirname(sourceFile.path),
-              path.join(base, "src", "index"),
-            );
-            console.log(
-              `[dev-tool/register] Rewrote import of "${oldName}" to "${node.moduleSpecifier.text}".`,
-            );
-          }
-          return node;
-        },
-        transformationContext,
-      ),
-  ],
-});
+let tsconfigPath;
```

Comment:
Would this be an error if we couldn't find the tsconfig?

Reply:
You know, it's expected for min/max when we're actually running the tests from one level deep but unexpected otherwise. Wasn't sure if there's a way to tell we're in a min/max test scenario... we could try it in a follow-up PR though - I need to look into min/max anyway I just want to avoid boiling the ocean in this PR. I can create an issue for it